packages/react-divider/Spec.md

```diff
@@ -190,11 +78,12 @@ Border Color: tokens.alias.color.neutral.neutralStroke1
 ## Behaviors
 
 This component has no state.
-This control will have no interactions.
-This control will have no effect on screen readers.
 
-Content, if provided, will self determine it's behaviors.
+This component has no mouse, touch or keyboard interactions.
+
+Content, if provided, will self-determine its behaviors.
 
 ## Accessibility
 
-Since the divider is not a focusable element itself, this control doesn't require any accessibility roles or special behavior applied to it. The descendant content when wrapped from the divider must handle any required accessibility behaviors itself when appropriate.
+- The `Divider` component should be assigned a `role="separator"` by default.
+- The `Divider` component should have no other accessibility concerns otherwise. The descendant content in the divider must handle any required accessibility behaviors itself when appropriate.
```

Comment:
Should we specify that the divider should be named by its content?

Reply:
We should! Let me add that bullet point.